package.json

```diff
@@ -54,7 +56,8 @@
     "reactstrap": "5.0.0-alpha.4",
     "redux": "3.7.2",
     "redux-debounce": "1.0.1",
-    "taskcluster-client": "2.5.4"
+    "taskcluster-client": "2.5.4",
+    "taskcluster-client-web": "^5.0.1"
```

Comment:
Could you make this a fixed version rather than a tilde range? (Renovate bumps the versions for us, so we don't use tilde ranges. The `.yarnrc` in the repo root is configured with `save-exact true` - did you use npm/add manually instead?).

Reply:
I used `yarn add taskcluster-client-web`.

Reply:
What version of yarn? (`save-exact` support was added in 1.1.0, in yarnpkg/yarn#4471)

Reply:
`1.3.2` :)